api/nedluzimstatu/ovm.ts

```diff
@@ -0,0 +1,34 @@
+import { createProxyMiddleware } from "http-proxy-middleware";
+import { NowRequest, NowResponse } from "@now/node";
+import { addCORSHeaders, runMiddleware } from "./_shared";
+import * as express from "express";
+
+const apiKey = process.env.APITALKS_API_KEY as string;
+const proxyTimeout = 8000;
+
+const ovmApi = createProxyMiddleware({
+  target: "https://api.apitalks.store/ovm",
```

Comment:
Tohle je 404, bude potřeba najít správný endpoint.

Reply:
endpoint je dobře, viz (https://www.api.store/organy-verejne-moci/), ale úplně jsem nepochopil mapování cesty z requestu na proxyrequest (i bez vercelu se mi tam jednou kus původní cesty zkopíruje a podruhé ne). veřejná cesta k api musí nabízet obojí, např. to /ovm a /spadovost

Reply:
A je nutný používat ten proxy middleware? Nejde prostě přebrat parametry od klienta, předat je „ručně“ ApiTalks a výsledek zase předat zpátky klientovi?

Reply:
@zoul úplně nutné to není, pokud něco takového umí vercel, můžeme použít. Přepisovat hotovou proxy knihovnu mi ale moc smysl nedává. (dělají to takhle i jinde, např. https://github.com/vercel/next.js/discussions/14057)

Reply:
@zoul podařilo se k tomu něco najít? v kódu mi nějak chybí definice zdrojové URI, která se dá přes http volat - kde se nastavuje? aneb neměl by být nějaký další rewrite i v now.json?